melosys-eessi-app/src/main/resources/application.yml

```diff
@@ -88,3 +88,4 @@ no.nav.security.jwt.issuer:
   isso:
     discovery-url: ${ISSO_URL}/.well-known/openid-configuration
     accepted-audience: ${ISSO_ACCEPTED_AUDIENCE}
+    cookie-name: ID_token
```

Comment:
Skal denne være med?

Reply:
Ja - når man logger inn i melosys, så settes et http-only cookie i nettleseren, med navn `ID_token`. Med denne linjen konfigurert, så slipper man å paste inn oidc-token i swaggeren, men kan bruke sesjonen man har med seg fra melosys 🙂

Reply:
...og siden vi ønsker å bruke swaggeren her i prod for å kunne kopiere/generere opp nye BUCer blir det mye greiere å bruke swaggeren 😄